### DISCUSSION TITLE
Can SignServer be used with PGP to sign JARs for publishing on Maven Central?

### DISCUSSION THREAD

**Discussion by ralienpp** [Keyfactor/signserver-ce#21] [Q&A] [answered]
My understanding is that a JAR will only be accepted by Maven Central of it was [signed with PGP](https://central.sonatype.org/publish/requirements/gpg/). 

1. Is there a way to use SignServer such that its signatures would be accepted by Maven Central?
1. Do you use SignServer to handle the keys used signing your own artifacts for Maven Central? (Having looked around signserver.org, I see that your own artifacts are [signed with OpenPGP, that relies on RSA keys](https://www.signserver.org/signserver-keys/)).

**Comment by ralienpp** [comment, marked as answer]
For the benefit of others looking for the same thing, the docs have a section about OpenPGP: https://doc.primekey.com/signserver/signserver-operations/setting-up-openpgp-signer

**Comment by netmackan** [comment]
1. Yes, the OpenPGP ascii armored detached signatures created by the OpenPGPSigner in SignServer can be used. (As you found out in the second post)
2. We started signing the release artifacts and artifacts for the Central repository before we had that functionality in SignServer and still it is done with a manual process but I really like to start using SignServer for that obviously :)
